n3fit/setup.py

```diff
@@ -13,6 +13,7 @@
 
         entry_points = {'console_scripts':
             ['n3fit = n3fit.scripts.n3fit_exec:main',
+             'evolven3fit_new = n3fit.scripts.evolven3fit_new:main',
```

Comment:
Is this going to be the final name of the thing?

Reply:
I hope not but for the moment we want to keep also the old `evolven3fit` so this is a temporary name. The moment we remove the old one we are going to call it just `evolven3fit`.